### DISCUSSION TITLE
Create a way to "Add Description" to multiple selected photos at once

### DISCUSSION THREAD

**Discussion by gitpitt06** [ente-io/ente#642] [Enhancements]
### Suggestion 
- As a user, I want to select multiple photos and "add a description." The description would then be added to _all_ selected photos. 
- Depending on complexity, maybe this could be enhanced to give users a choice: "add description to _all_ selected photos" or "add description _only_ to selected photos that have an empty description."

This is similar to being able to select multiple photos and "fix time." 

I was not sure what repo to suggest this in, but in addition to the desktop app this could help on the web (https://github.com/ente-io/photos-web/issues)

### Context
Suppose I am uploading old photos from my computer to ente. The photos may have been taken with an old camera, may be missing metadata, etc. As an example, suppose I want to upload a folder called "Winter 2008."
- In "Winter 2008" folder, there are two folders: "London" and "Paris"
- "London" has 100 photos and "Paris" has 100 photos

Currently I can easily import "Winter 2008" and end up with two albums, one for "London" and one for "Paris." I then want to:
- select all 100 photos in "London" and add a description of "Photo taken in London, England"
- select all 100 photos in "Paris" and add a description of "Photo taken in Paris, France"
- finally, after adding a description to all 200 photos, I want to merge "London" album and "Paris" album into a single album (this is possible already)

The issue is that I do not think I can import "Winter 2008" as a single album of 200 photos because I would lose information about which 100 photos are from each city. The folder names on my computer contain information about the photos, but if I upload thousands of photos at once - and have every folder turned into an album to capture that information - I end up with a lot of albums. Being able to add a description and then merge albums helps solve this.

**Comment by quietmansoath** [comment]
+1 for this
